third_party/2and3/attr/validators.pyi

```diff
@@ -1,9 +1,9 @@
-from typing import Container, List, Union, TypeVar, Type, Any, Optional
+from typing import Container, List, Union, TypeVar, Type, Any, Optional, Sequence
 from . import _ValidatorType
 
 _T = TypeVar('_T')
 
-def instance_of(type: Type[_T]) -> _ValidatorType[_T]: ...
+def instance_of(type: Union[Sequence[Type[_T]], Type[_T]]) -> _ValidatorType[_T]: ...
```

Comment:
Shouldn't it be specifically a `Tuple[Type[_T], ...]`? `isinstance(x, [int, str])` doesn't work.

Reply:
Right. I was trying to use `Tuple`, but I didnt know about the syntax with three dots (`Tuple[X, ...]`).

Fixed now, thanks.